web/packages/core/src/ruffle-player.ts

```diff
@@ -1421,10 +1458,25 @@ export class RufflePlayer extends HTMLElement {
         }\n`;
     }
 
+    private hidePreloader(): void {
+        this.preloader.style.display = "none";
+        this.unmuteOverlay.classList.remove("hidden");
+        this.playButton.classList.remove("hidden");
+    }
+
+    private showPreloader(): void {
+        this.preloader.style.display = "flex";
```

Comment:
I think this can be replaced with:
```ts
this.preloader.classList.remove("hidden");
```
And the corresponding line in `hidePreloader()` can be replaced with:
```ts
this.preloader.classList.add("hidden");
```